backend-rust/src/indexer.rs

```diff
@@ -3639,7 +3733,10 @@ async fn process_cis2_token_event(
             )
             .execute(tx.as_mut())
             .await?
-            .ensure_affected_one_row()?;
+            .ensure_affected_one_row()
+            .with_context(|| {
+                format!("Failed inserting the token transfer event: {:?}", cis2_transfer_event)
+            })?;
```

Comment:
The `context` should be on the `await` error since this is the action that inserts.

```
  .execute(tx.as_mut())
                .await
                .context("Failed inserting the token transfer event: {:?}", cis2_transfer_event")?
                .ensure_affected_rows_in_range(0..=1)?;
```

Reply:
I'll add it to both then, the current is on the validation of the affected rows so also related to the action that inserts.
Ideally, we would only have this context once on a database operation struct so it covers both cases.